[en]: Update ghippo v0.31.0 release notes (#5643)
DaoCloud/DaoCloud-docs: docs/en/docs/ghippo/intro/release-notes.md

This page lists the Release Notes for global management of each version,
so that you can understand the evolution path and feature changes of each version.

## 2024-9-30

### v0.31.0

#### Features

- **Added** support for username encryption when creating users
- **Added** custom roles that support Container Management folders and workspace roles, and allow their permission points to be mapped to predefined roles in Container Management

#### Fixes

> - **Fixed** an error issue when the license expiration time is long.

Reviewer: ```suggestion
- **Fixed** an error related to excessively long license expiration periods.
```

> - **Fixed** an error issue when unbinding resources from a workspace.

Reviewer: ```suggestion
- **Fixed** an error encountered when a resource is unbound from a workspace.
```
